core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationDirective.js

```diff
@@ -285,6 +287,50 @@ oppia.directive('stateTranslation', [
             $scope.activeHintIndex = null;
             $scope.activeAnswerGroupIndex = null;
 
+            var currentCustomizationArgs = ExplorationStatesService
+              .getInteractionCustomizationArgsMemento(stateName);
+            var interactionId = $scope.stateInteractionId;
+            if (interactionId) {
+              var interactionPreviewHtml = ExplorationHtmlFormatterService
```

Comment:
is this an unused var?

Reply:
Also if it's an unused var then we remove the unused dependencies too. (`ExplorationHtmlFormatterService`)

Reply:
I'm not sure why we are using `$rootScope.$broadcast('updateAnswerChoices'` 4 times in StateInteractionEditorDirective.js line 424, is there any reson behind following that pattern?

Reply:
I think it was because the value to be broadcasted is different in the 4 cases, I think it would be possible to store it in a variable and just broadcast after the if - else conditions, though not sure if that's necessary.